.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,50 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+	"name": "Rust",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/rust:1-1-bullseye",
```

Comment:
can you use `rust:latest`? I think the newer jolt toolchain based on rust 1.86.0 needs glibc 2.32+ and GLIBCXX_3.4.29+ which your currently selected version doesn't meet.

Reply:
I bump it from bullseye to bookworm, it should meet your requirements.
```
> ldd --version
ldd (Debian GLIBC 2.36-9+deb12u10) 2.36

> strings /usr/lib/aarch64-linux-gnu/libstdc++.so.6 | grep GLIBCXX_3.4.3  
GLIBCXX_3.4.3
GLIBCXX_3.4.30
```